test/test_cores/test_fifo/test_fifo_ramp.py

```diff
@@ -19,8 +19,13 @@
 
 from rhea.utils.test import run_testbench, tb_args
 
+# temp, due to pytest 10x runtime error.
+long_test = pytest.mark.skipif(
+    not pytest.config.getoption("--runslow"),
+    reason="need --runslow option to run"
+)
 
-@pytest.mark.skipif(True, reason="pytest issue/error 10x runtime")
+@long_test
```

Comment:
If you are proposing to use the short-hand `@long_test` it needs to be put somewhere accessible by all tests and not repeated (DRY) in each test.  It can be put in `rhea.utils.test` 

`from rhea.utils.test import long_test`